affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleConfiguration.kt

```diff
@@ -66,7 +67,8 @@ class AffectedModuleConfiguration {
             requireNotNull(baseDir) {
                 "baseDir must be set to use pathsAffectingAllModules"
             }
-            field = value
+            // Protect against users specifying the wrong path separator for their OS.
+            field = value.map { it.toOsSpecificPath() }.toSet()
```

Comment:
In a project with multiple developers you can very easily have one developer specify `paths/affecting/all/modules` and another specifying `path\to\all\modules` because that is what is natural on their host OS. However we want to sanitise all of that and be consistent for the OS.